src/chi/components/alert/_banner.scss

```diff
@@ -6,63 +6,21 @@
       border-radius: 0;
       border-width: 0.0625rem 0;
       padding-left: 1rem;
-      padding-right: 2rem;
+      padding-right: 1rem;
 
-      &.-small {
-        display: flex;
-        justify-content: center;
-        padding-left: 2rem;
-
-        &::before {
-          display: none;
-        }
+      &.-dismiss {
+        justify-content: space-between;
       }
 
       &.-large {
         padding-bottom: 1.1875rem;
```

Comment:
@davidojedacl @jllr Padding-bottom on -large alerts is currently set to 19px. Let's change to the same value as the top, `.75rem`

Reply:
fixed